src/api/app/models/user.rb

```diff
@@ -24,6 +24,10 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy, inverse_of: :user
   has_many :status_messages
   has_many :tokens, class_name: 'Token', dependent: :destroy, inverse_of: :user
+  
+  has_many :workflow_token_users, dependent: :destroy
+  has_many :workflow_tokens, class_name: 'Token::Workflow', through: :workflow_token_users, source: :token
```

Comment:
Why "has many through"? What are the plans with the in-between model?

Sounds to me like [has and belongs to many](https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association) should be enough, no?